packages/registry/docs/conceptual-guides/add-on-loader.md

```diff
@@ -0,0 +1,97 @@
+---
+myst:
+  html_meta:
+    "description": "An explanation of the add-ons configuration loader in @plone/registry"
+    "property=og:description": "An explanation of the add-ons configuration loader in @plone/registry"
+    "property=og:title": "Add-ons configuration loader"
+    "keywords": "@plone/registry, registry, add-ons, loader"
+---
+
+# Add-ons configuration loader
+
+The `@plone/registry` compatible add-ons are able to load configuration into the configuration registry.
+This is achieved by using the `main` key entry point specified in the add-on {file}`package.json`.
```

Comment:
```suggestion
The value of the `main` key entry point in the add-on {file}`package.json` loads configuration.
```

Reply:
not really, this could be confusing.
So, @plone/registry reads this value to know the source of the loader, not to load anything.

Reply:
Suggestion updated. One more review?